core/Plan.go

```diff
@@ -39,3 +41,47 @@ func (instance Plan) AddJob(job Job) Plan {
 	instance.Jobs = jobs
 	return instance
 }
+
+//Lists returns the configured lists for the plan
+func (instance Plan) Lists() map[string][]map[string]interface{} {
+	var lists = map[string][]map[string]interface{}{}
+
+	if instance.Context["lists"] != nil {
+		listKeys := instance.Context["lists"].(map[interface{}]interface{})
+		for listKey, listValue := range listKeys {
+			lists[listKey.(string)] = []map[string]interface{}{}
+			listValueItems := listValue.([]interface{})
+			for _, listValueItem := range listValueItems {
+				srcData := listValueItem.(map[interface{}]interface{})
+				stringKeyData := map[string]interface{}{}
+				for srcKey, srcValue := range srcData {
+					stringKeyData[srcKey.(string)] = srcValue
+				}
+				lists[listKey.(string)] = append(lists[listKey.(string)], stringKeyData)
+			}
+		}
+	}
+	return lists
+}
+
+//Clone returns a clone of the current plan
+func (instance Plan) Clone() Plan {
+
+	var mod bytes.Buffer
+	enc := gob.NewEncoder(&mod)
+	dec := gob.NewDecoder(&mod)
+
+	err := enc.Encode(instance)
+	if err != nil {
+		panic(err)
```

Comment:
This is an oversight.  Good catch again.  I have now deleted this code as nothing is using it.  This was a failed attempt at how I was trying to achieve the ticket.